workflow-examples/src/test/java/com/redhat/parodos/examples/integration/utils/ExamplesUtils.java

```diff
@@ -104,4 +110,44 @@ private static class AsyncResult {
 
 	}
 
+	public static ProjectResponseDTO commonProjectAPI(ApiClient apiClient, String projectName,
```

Comment:
@gciavarrini this function include common code also used in the `ExampleWorkFlow`
After this PR will be merged I will open another PR changing `ExampleWorkFlow` to use this function

Reply:
Please check (and review :) ) this [commit](https://github.com/parodos-dev/parodos/pull/240/commits/078dfe649c4b16bd8388ebb00094ee7c5f077c2e) 

If my change is ok, then I think it's better to use the new implementation.

Reply:
Ignore my previous comment. 
This change is different and not related to my PR.